test/test_crud.py

```diff
@@ -81,7 +84,7 @@ def run_playbook(module, extra_vars=None, limit=None):
 
 
 @pytest.mark.parametrize('module', MODULES)
-def test_crud(module, record):
-    run = run_playbook_vcr(module, record=record)
+def test_crud(tmpdir, module, record):
```

Comment:
Is tmpfile a pytest fixture, or do we need to configure that somewhere?

Reply:
It's a pytest fixture, yeah. https://docs.pytest.org/en/latest/tmpdir.html

We don't need to do anything. 

But I can also drop that change from this PR if you want to and we'll discuss it separately.

Reply:
If pytest handles tmpdirs for us, that's fine. :+1: